easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-2.3.1-intel-2015a-eb-deps.eb

```diff
@@ -0,0 +1,51 @@
+name = 'OpenFOAM'
+version = '2.3.1'
+versionsuffix = '-eb-deps'
+paraversion = '4.3.1'
+pyver = '-Python-2.7.10'
```

Comment:
it pulls in Python via two of its deps, so I feel it should be mentioned in versionsuffix?